### ISSUE TITLE
G3X Touch events mislabeled

### ISSUE THREAD

**Issue by davidlav** [MobiFlight/HubHop-Website#54]
**Describe the bug**
There are 12 events that support the stock G3X Touch display (used in the X Cub and VL-3). 11 of them are erroneously prefixed with an underscore.

**Steps To Reproduce**
See lines `1666 - 1676` of `events.txt` (below).  You'll see they begin with an underscore character.

https://github.com/MobiFlight/MobiFlight-Connector/blob/dc5772bdf38e63c91d8d30512109913e37f6ce1e/MSFS2020-module/mobiflight-event-module/modules/events.txt#L1666-L1676

**Expected behavior**
They should instead look like the one correctly-name event, orphaned by itself on line `1865` (below).

https://github.com/MobiFlight/MobiFlight-Connector/blob/dc5772bdf38e63c91d8d30512109913e37f6ce1e/MSFS2020-module/mobiflight-event-module/modules/events.txt#L1865

**Additional context**
Hubhop has them named correctly, without prefixed underscores:

![image](https://user-images.githubusercontent.com/18689323/141870055-b9df2710-f676-4d17-9187-cf69829ba34a.png)

**Comment by MobiFlight-Admin** [comment]
I moved the issue to the Hubhop Repository

**Comment by rofl-er** [comment]
Thanks. Can you check what you can do in the exporter, @MobiFlight-Admin ?

**Comment by JaimeLeon2** [comment]
I think the correct solution is to not allow the event names to start or end with spaces or any special characters, only alpha numeric, when the events are created, rather than patching it on export.

**Comment by MobiFlight-Admin** [comment]
I agree, sanitizing input should be done on saving... but even there it could be on API level or the front end or both.

API is probably the better spot because we can submit stuff without UI too

**Comment by rofl-er** [comment]
Leading/Trailing Spaces are now removed and only RPN Syntax is allowed in the code. Special characters are removed from the label.

**Comment by rofl-er** [comment]
@MobiFlight-Admin - close the issue if you are happy with it
